ontology/yaml/resources/units/units.yaml

```diff
@@ -32,27 +45,83 @@ density:
 - grams_per_cubic_meter
 - micrograms_per_cubic_meter
 - milligrams_per_cubic_meter
+electriccapacitance:
+- farads: STANDARD
+electricconductance:
+- siemens: STANDARD
+electricconductivity:
+- siemens_per_meter: STANDARD
+electriccurrentdensity:
+- amperes_per_square_meter: STANDARD
+electricfield:
+- volts_per_meter: STANDARD
+electricinductance:
+- henrys: STANDARD
+electricphasedifference:
+- degrees_phase: STANDARD
+electricfieldstrength:
+- amperes_per_meter: STANDARD
+electricresistivity:
+- ohm_meters: STANDARD
 energy:
 - joules: STANDARD
 - kilowatt_hours
+- btus
+- kilobtus
+- kilojoules
+- megabtus
+- megajoules
 - megawatt_hours
+- therms
+- ton_hours
+- watt_hours
+energydensity:
+- kilojoules_per_kilogram: STANDARD
+entropy:
+- joules_per_degree_kelvin: STANDARD
```

Comment:
nit: technically kelvin is not in degrees.

Reply:
Thanks for spotting this, done